StockTrader/src/main/java/com/restResource/StockTrader/controller/SellController.java

```diff
@@ -45,61 +46,55 @@ Quote createNewSell(
             @RequestParam String userId,
             @RequestParam String stockSymbol,
             @RequestParam int amount) {
-
-        loggingService.logUserCommand(
-                UserCommandLog.builder()
-                        .username(userId)
-                        .stockSymbol(stockSymbol)
-                        .funds(amount)
-                        .command(CommandType.SELL)
-                        .build());
-
-        if (amount <= 0) {
-            throw new IllegalArgumentException(
-                    "The amount parameter must be greater than zero.");
-        }
-
         Quote quote = quoteService.getQuote(stockSymbol, userId);
+        try {
+            //loggingService.logUserCommand(CommandType.SELL,userId,stockSymbol,null,amount);
```

Comment:
was there a problem having the amount here?

Reply:
no. amount added.